### ISSUE TITLE
ERR_MODULE_NOT_FOUND esbuild

### ISSUE THREAD

**Issue by XMadFoX** [drizzle-team/drizzle-kit-mirror#94]
Hi, when upgrading from 0.18 to 0.19 I get this error when executing any drizzle kit command:
```
drizzle-kit push:sqlite

node:internal/process/esm_loader:97
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '/path/to/project/node_modules/drizzle-kit/node_modules/esbuild/' imported from /path/to/project/node_modules/drizzle-kit/loader.mjs
Did you mean to import esbuild/lib/main.js?
    at new NodeError (node:internal/errors:399:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:235:9)
    at packageResolve (node:internal/modules/esm/resolve:876:14)
    at moduleResolve (node:internal/modules/esm/resolve:938:20)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```
<details>
<summary>System, version details</summary>
OS: Linux 6.3
CPU: x64
Node: 18.16.0 
npm: 9.5.1 
drizzle-orm: 0.27.0
drizzle-kit: 0.19.1

</details>

Something similar discussed [here](https://github.com/drizzle-team/drizzle-kit-mirror/issues/55#issuecomment-1597338432)
I also tried `@win` version [0.19.2-b928697](https://www.npmjs.com/package/drizzle-kit/v/0.19.2-b928697) and got:
```
/usr/bin/env: ‘node --loader @esbuild-kit/esm-loader --loader drizzle-kit/loader.mjs --no-warnings’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

**Comment by michaelschufi** [comment]
I can confirm that the same happens on WSL with the `@win (0.19.2-b928697)`:

```node
> drizzle-kit generate:pg

node:internal/process/esm_loader:97
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@esbuild-kit/esm-loader' imported from /path/to/project/
Did you mean to import @esbuild-kit+esm-loader@2.5.5/node_modules/@esbuild-kit/esm-loader/dist/index.js?
    at new NodeError (node:internal/errors:400:5)
    at packageResolve (node:internal/modules/esm/resolve:889:9)
    at moduleResolve (node:internal/modules/esm/resolve:938:20)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:842:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:525:22)
    at initializeLoader (node:internal/process/esm_loader:75:58)
    at loadESM (node:internal/process/esm_loader:90:11) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

**Comment by gjedlicska** [comment]
After upgrading to the `win` tag i get a different issue:

```
❯ yarn run drizzle-kit generate:pg --config=drizzle.config.ts
drizzle-kit: v0.19.2
drizzle-orm: v0.27.0

Reading config file '/home/gergojedlicska/Speckle/speckle-automate/components/automate-app/drizzle.config.ts'
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/gergojedlicska/Speckle/speckle-automate/components/automate-app/drizzle.config.ts__drizzle__' imported from /home/gergojedlicska/Speckle/speckle-automate/components/automate-app/node_modules/drizzle-kit/index.cjs
    at new NodeError (node:internal/errors:399:5)
    at finalizeResolution (node:internal/modules/esm/resolve:326:11)
    at moduleResolve (node:internal/modules/esm/resolve:945:10)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:525:22)
    at importModuleDynamically (node:internal/modules/cjs/loader:1186:29)
    at importModuleDynamicallyWrapper (node:internal/vm/module:429:21) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.16.0
```

**Comment by hanssonduck** [comment]
A workaround in the meantime is to move all schema definitions to the same file and not import any other files from it.

**Comment by AndriiSherman** [comment]
Explained a fix [here](https://github.com/drizzle-team/drizzle-orm/issues/781#issuecomment-1600679308)

Will be in latest today

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.2`
